processing/src/main/java/org/apache/carbondata/processing/loading/parser/impl/StructParserImpl.java

```diff
@@ -59,6 +59,14 @@ public StructObject parse(Object data) {
           }
           return new StructObject(array);
         }
+      } else if (value.isEmpty()) {
```

Comment:
I just checked the code , **Need to handle the same for `MapParserImpl` also**. where as **ArrayParserImpl** is already handled this.